pyInterface/package/_fileManager.py

```diff
@@ -108,187 +136,144 @@ def initialize(self, configObject):
 		return True
 
 
-	def getMissingBins(self):
-		missingBins = []
-		for binID in self.getBinIDList():
-			currentBin = self.binList[binID]
-			for eventsType in self.dataFiles:
-				found = False
-				for dataFile in self.dataFiles[eventsType]:
-					if dataFile.binningMap == currentBin:
-						found = True
-						break
-				if not found:
-					missingBins.append(binID)
-		return missingBins
-
-
-	def getDataFilePaths(self):
-		return fileManager.convertDataFilesToPaths(self.dataFiles)
-
-
-	def getDataFile(self, binID, eventsType):
-		internalEventsType = fileManager.eventsTypeFromBpEnum(eventsType)
-		if not internalEventsType in self.dataFiles:
-			pyRootPwa.utils.printErr("did not find data file with eventsType '" + str(eventsType) + "'.")
-			return None
-		for dataFile in self.dataFiles[internalEventsType]:
-			found = True
-			if not dataFile.binningMap.keys() == self.binList[binID].keys():
-				continue
-			for binningVariable in dataFile.binningMap:
-				if not self.binList[binID][binningVariable] == dataFile.binningMap[binningVariable]:
-					found = False
-					break
-			if found:
-				return dataFile
-		pyRootPwa.utils.printWarn("no data dataFile found for binID = " + str(binID) + "and eventsType = '" + str(eventsType) + "'.")
-		return None
-
+	def checkBinning(self):
+		raise Exception("not implemented. " + repr(self))
```

Comment:
or simply return `True` for the moment, but call the function at the appropriate places